File: src/main/java/org/jetbrains/fortran/lang/psi/impl/FortranStmtImpl.java
package org.jetbrains.fortran.lang.psi.impl;

import com.intellij.lang.ASTNode;
import org.jetbrains.fortran.lang.psi.FortranStmt;

public class FortranStmtImpl extends FortranCompositeElementImpl implements FortranStmt {
    public FortranStmtImpl(ASTNode node) {
        super(node);
    }
}

File: src/main/java/org/jetbrains/fortran/lang/psi/impl/FortranStmtImpl.kt
package org.jetbrains.fortran.lang.psi.impl

import com.intellij.lang.ASTNode
import org.jetbrains.fortran.lang.psi.FortranStmt

open class FortranStmtImpl(node: ASTNode?) : FortranCompositeElementImpl(node!!), FortranStmt
